tests/llm/fixtures/test_ask_holmes/212_large_configmap_needle/generate_configs.py

```
@@ -0,0 +1,240 @@
+        "MAX_CONNECTIONS": str(rng.randint(10, 200)),
+        "WORKER_THREADS": str(rng.randint(1, 16)),
+        "GRACEFUL_SHUTDOWN_TIMEOUT": f"{rng.randint(5, 30)}s",
+        "REQUEST_BODY_LIMIT": f"{rng.choice([1, 5, 10, 50])}mb",
+    }
+
+    lines = []
+    lines.append(f"  {name}:")
+    lines.append(f"    name: {name}")
+    lines.append(f"    version: \"{version}\"")
+    lines.append(f"    team: {team}")
+    lines.append(f"    deployment:")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)